OO.Rmd

```diff
@@ -150,3 +150,19 @@ otype(mle_obj)
 ```
 
 Use this function to figure out which chapter to read to understand how to work with an existing object.
+
+
+1.  Chapter \@ref(base-types) first details the basic types upon which the more
```

Comment:
I think this should come after the paragraph starting "The goal of this brief introductory chapter"

Reply:
Changed!